cli/src/main/resources/help/server.adoc

```diff
@@ -0,0 +1,67 @@
+SERVER(1)
+=========
+:doctype: manpage
+
+
+NAME
+----
+server - server configuration and state management.
+
+
+DESCRIPTION
+-----------
+
+
+
+SYNOPSIS
+--------
+*server report*
+
+*server connector ls*
+
+*server connector describe* 'connector-name'
+
+*server connector start* 'connector-name'
+
+*server connector stop* 'connector-name'
+
+*server connector ipfilter ls* 'connector-name'
+
+*server connector ipfilter set* 'connector-name' --rules='[ACCEPT|REJECT]/cidr',...
+
+*server connector ipfilter clear* 'connector-name'
+
+
+SERVER CONNECTOR IPFILTER OPTIONS
+---------------------------------
+
+*--rules*='[ACCEPT|REJECT]/cidr',...::
+One or more IP filtering rules.
+
+
+EXAMPLES
+--------
+
+`server report` +
+Obtains a server report, including information about network, threads, memory, etc.
+
+`server connector ls` +
+Lists all available connectors on the server.
+
+`server connector describe endpoint-default` +
+Shows information about the specified connector, including host, port, local and global connections, IP filtering rules.
+
+`server connector start endpoint-default` +
```

Comment:
I guess this is a bad example. if `endpoint-default` is stopped, the cli can't connect.

Reply:
I took the liberty of changing this to `my-hotrod-connector` because users will mostly likely not be using this command with default connectors.